core/src/main/scala/org/apache/spark/rdd/ReliableRDDCheckpointData.scala

```diff
@@ -55,25 +55,7 @@ private[spark] class ReliableRDDCheckpointData[T: ClassTag](@transient private v
    * This is called immediately after the first action invoked on this RDD has completed.
    */
   protected override def doCheckpoint(): CheckpointRDD[T] = {
-
```

Comment:
All this code has been moved in the `ReliableCheckpointRDD.createCheckpointedRDD`